base/error.jl

```diff
@@ -243,7 +245,7 @@ struct ExponentialBackOff
     jitter::Float64
 
     function ExponentialBackOff(n, first_delay, max_delay, factor, jitter)
-        all(x->x>=0, (n, first_delay, max_delay, factor, jitter)) || error("all inputs must be non-negative")
+        all(x->x>=0, (n, first_delay, max_delay, factor, jitter)) || throw(ArgumentError("all inputs must be non-negative"))
```

Comment:
This should be a `DomainError`
```suggestion
        all(x->x>=0, (n, first_delay, max_delay, factor, jitter)) || throw(DomainError((;n, first_delay, max_delay, factor, jitter)[findall(<(0), (n, first_delay, max_delay, factor, jitter))] ,"all inputs must be non-negative"))
```